src/features/auth/store/auth.actions.ts

```diff
@@ -43,7 +43,7 @@ const signin = createAsyncThunk<string, string, { rejectValue: void }>(
     const token = await authService.signin(signature, nonce);
 
     localStorageService.setItem(constants.tokenKey, { access_token: token });
-
+    window.location.reload();
```

Comment:
Why are you reloading the page here ?

Reply:
If I don't reload the page, the useEffect can't catch up the localStorage change, so the button can't be rendered

Reply:
not sure if I get it correctly, but don't use a state variable instead? If you refresh the page, you are getting rid of all the redux state and you are making the ux to behave annoying.